core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -775,11 +826,17 @@ public PersistenceState getPersistenceState(long fileId) throws FileDoesNotExist
       loadMetadataIfNotExistAndJournal(inodePath, loadMetadataOptions, journalContext);
       ensureFullPathAndUpdateCache(inodePath);
       inode = inodePath.getInode();
+      auditContext.setSrcInode(inode);
```

Comment:
When we create the audit context, we should set the src inode to `inodePath.getInodeOrNull()`